packages/cardpay-sdk/sdk/utils/safe-utils.ts

```diff
@@ -75,6 +86,35 @@ export interface GnosisExecTx extends RelayTransaction {
   transactionHash: string;
 }
 
+export async function createSafe(
+  web3: Web3,
+  owners: string[],
+  threshold: number,
+  saltNonce: string,
+  paymentToken: string
+): Promise<CreateSafe> {
+  let relayServiceURL = await getConstant('relayServiceURL', web3);
+  let url = `${relayServiceURL}/v3/safes/`;
+  let options = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Accept: 'application/json',
+    },
+    body: JSON.stringify({
+      owners,
+      threshold,
+      saltNonce,
+      paymentToken,
+    }),
+  };
```

Comment:
I don't think we can use this--this is _specifically_ for create2 safes. As @lukemelia mentioned chris won't like to not use create2 safes since we don't have a salt that we want to use (a "vary" that derives the conterfactual safe addresses). Rather we need to use non-create2 safes that don't have a salt as an input

Reply:
we probably need to  do this all using web3, we can just call the gnosis safe proxy factory to create a non-create2 safe, and then another txn to send our tokens into the newly created safe.

An alternative to this though could be that we make a custom contract whose job is to create a safe and then transfer tokens from the msg.sender to the safe. This would probably need to be a 2 txn though (because of erc-20 approve flow), as the first txn would need to set an allowance on this new contract, and the 2nd txn would be the txn that creates the safe and moves the tokens into the safe. since this is also 2 txns, not sure this would buy us much... if we had erc-677 tokens though, it would be just a single txn. but that is wishful thinking....